config/settings/tailer.q

```diff
@@ -0,0 +1,63 @@
+                                                                                            //                       at EOD the data will be merged from each partiton before being moved to hdb
+												
+mergenumrows:100000                                                                         // default number of rows for merge process
+mergenumtab:`quote`trade!10000 50000                                                        // specify number of rows per table
+
+tpconnsleepintv:10                                                                          // number of seconds between attempts to connect to the tp
+upd:insert                                                                                  // value of the upd function
+replay:1b                                                                                   // replay the tickerplant log file
+schema:1b                                                                                   // retrieve schema from tickerplant
+settimer:0D00:00:10                                                                         // timer to check if data needs written to disk
+partitiontype:`date                                                                         // set type of partition (defaults to `date, can be `date, `month or `year)
+gmttime:1b                                                                                  // define whether the process is on gmttime or not
+getpartition:{@[value;`.wdb.currentpartition;(`date^partitiontype)$(.z.D,.z.d)gmttime]}     // function to determine the partition value
+reloadorder:`hdb`rdb                                                                        // order to reload hdbs and rdbs
+hdbdir:`:hdb                                                                                // move wdb database to different location
+sortcsv:hsym first .proc.getconfigfile"sort.csv"                                            // location of csv file
+permitreload:1b                                                                             // enable reload of hdbs/rdbs
+compression:()                                                                              // specify the compress level, empty list if no required
+gc:1b                                                                                       // garbage collect at appropriate points (after each table save and after sorting data)
+eodwaittime:0D00:00:10.000                                                                  // time to wait for async calls to complete at eod
+tpcheckcycles:0W                                                                            // number of attempts to connect to tp before process is killed
+
+\d .ds
+datastripe:0b
+period:0D01
+lasttimestamp:.z.p-4*period
```

Comment:
I don't think this should be calced here - I think there should maybe be a config value for number of periods, but calc the lasttimestamp in tailer.q, using the relevant config variables

On a side note, we need to make sure it's well documented all the places that things like period & number of periods etc. is configured.